homeassistant/components/knx/__init__.py

```diff
@@ -134,9 +135,7 @@
 SERVICE_KNX_SEND_SCHEMA = vol.Schema(
     {
         vol.Required(SERVICE_KNX_ATTR_ADDRESS): cv.string,
-        vol.Required(SERVICE_KNX_ATTR_PAYLOAD): vol.Any(
-            cv.positive_int, [cv.positive_int]
-        ),
+        vol.Required(SERVICE_KNX_ATTR_PAYLOAD): cv.match_all,
```

Comment:
Is this change required for the library bump?

Reply:
Its a bugfix for #44792
Should I do an extra PR for it?

Reply:
Yes, definitely move it to a separate PR. We want to be able to release fixes in patch releases. We can't do that easily if we mix the fix with other feature changes.

Reply:
Alright. Reverted.